portfolio/src/main/webapp/index.html

```diff
@@ -168,28 +168,36 @@ <h4 class="experience_header">Past Positions</h4>
 
                 <p>Name: </p>
                 <input type="text" name="comment_name">
-                <!-- <p>Comment: <input type="text" class="long_text" name="comment_content"></p> -->
                 <p>Comment: </p>
                 <textarea type="text" name="comment_content" rows="2"></textarea>
 
                 <input type="submit" />
             </form>
             <hr>
-            <!-- <div id="comments_container"> -->
-                <!-- <ul id="comment-list">
-                    <li class="comment"><span><strong>Andrew Hojel</strong></span><button class="comment_button delete"><i class="fa fa-trash"></i></button><br/><span>Let's try this Let's try this Let's try this Let's try this Let's try this Let's try thisLet's try this Let's try thisLet's try this vLet's try this</span></li> 
-                </ul> -->
-            <!-- </div> -->
+            <div>
+                <label for="comment_count">Number of Comments Displayed: </label>
+                <select name="comment_count" id="comment_count" onchange="getComments()">
```

Comment:
probably too late to worry about now, but for html we typically do kebab-case (not snake_case) for ids, classes, and pretty much everything else. you can feel free to clean this up (to be comment-count in this case) in a new PR or just skip it entirely and keep it in mind for the capstone. 
consistency is more important than style guides (usually! not always - just because someone did a weird thing once doesn't mean you should perpetuate it), so if you do make this change, please change everything.